airflow/models/dagbag.py

```diff
@@ -391,6 +391,10 @@ def collect_dags(
             'dagbag_import_errors', len(self.import_errors), 1)
         self.dagbag_stats = sorted(
             stats, key=lambda x: x.duration, reverse=True)
+        for file_stat in self.dagbag_stats:
+            Stats.timing('dag.loading-duration.{}'.
+                         format(str([dag.dag_id for dag in file_stat.dags])),
```

Comment:
Hi @feng-tao , based on lines https://github.com/apache/airflow/blob/65a2e1b66dd8b3f31f23579c2aaed13174e3316f/airflow/models/dagbag.py#L377-L383, `file_stat.dags` here is a **string** rather than a **list of string**. It may mean that your list comprehension here is not working as you intended?

Reply:
thanks @XD-DENG . You are right:) Somehow I thought the last field is a list. PR updated.